holoviews/core/util.py

```diff
@@ -73,6 +73,11 @@ class Config(param.ParameterizedFunction):
        recommended that users switch this on to update any uses of
        __call__ as it will be deprecated in future.""")
 
+    rtol = param.Number(default=10e-6, doc="""
```

Comment:
The config option should probably have a more specific name. Also it's not for all gridded data but specifically for Images.

Reply:
Sure. ``image_rtol`` then? Or do you have an alternative suggestion?